daxa/mission/xmm.py

```diff
@@ -260,6 +260,11 @@ def _download_call(observation_id: str, insts: List[str], level: str, filename:
         :return: A None value.
         :rtype: Any
         """
+        # The astroquery module will only download to the cwd, so need to change to where we want the data downloaded
+        og_dir = os.getcwd()  # storing this info so we can return here after the download is done
+        dest_dir_path = filename.replace(observation_id, '')  # this is a static method, so I cant use self.raw_data_path
+        os.chdir(dest_dir_path)  # changing to where we want the file downloaded
```

Comment:
this is my rudimentary solution to this problem. I wonder if we should also put in an issue to the astroquery github, I cant believe this is an intended behaviour? either way I have tested this and it gets rid of the error